### PR TITLE
Add docs for Prestodb to postgresql type mapping in postgresql connector

### PR DESCRIPTION
**Pull request by Akanksha-kedia**
##Description
This document aims to provide a comprehensive guide for PrestoDB to PostgreSQL type mapping within the PostgreSQL connector.

##Motivation and Context
The motivation behind this improvement is to enhance our PostgreSQL documentation. By providing clear type mapping details between PrestoDB and PostgreSQL, we can assist users in understanding how data types correlate and convert between these two systems.

##Impact
The main impact of this documentation is to provide clear and concise information about type mapping from PrestoDB to PostgreSQL and vice versa. This will help users in designing and implementing their data models and queries more efficiently and accurately.

##Test Plan
While there is no specific test plan for this documentation, the information provided has been validated through practical application. Tables were created in both PostgreSQL and PrestoDB, and the type mapping was cross-checked to ensure accuracy. Users are encouraged to perform similar tests to further familiarize themselves with the type mapping process.

[
Screenshot 2023-11-15 at 10 23 02 AM
Screenshot 2023-11-15 at 10 25 37 AM
Screenshot 2023-11-15 at 10 26 16 AM
Screenshot 2023-11-15 at 10 30 38 AM
Screenshot 2023-11-15 at 10 33 23 AM
](url)
## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

== NO RELEASE NOTE ==

### CODE REVIEW

**Comment by Akanksha-kedia** [comment]
@tdcmeehan , 
@steveburnett please help to review

File: presto-docs/src/main/sphinx/connector/postgresql.rst [hunk 1]
```diff
@@ -84,6 +84,109 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
...
+
+No other types are supported.
+
+PrestoDB type to PostgreSQL type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps PrestoDB types to the corresponding PostgreSQL types following
+this table:
+
+.. list-table:: PrestoDB type to PostgreSQL type mapping
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/postgresql.rst:155]:
```suggestion
.. list-table:: PrestoDB to PostgreSQL type mapping
```
Change suggested to parallel https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst

**Review by steveburnett — CHANGES_REQUESTED** [review]
Made suggestions to parallel your similar work in https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst.

Also, the tables on this page look squashed narrow compared to the same tables on  https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst , see screenshots. 

Screenshot 2023-11-15 at 1 05 05 PM
Screenshot 2023-11-15 at 1 05 09 PM

**Comment by Akanksha-kedia** [comment]
@steveburnett please check.

**Comment by Akanksha-kedia** [comment]
> Made suggestions to parallel your similar work in https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst.
> 
> Also, the tables on this page look squashed narrow compared to the same tables on https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst , see screenshots.
> 
> Screenshot 2023-11-15 at 1 05 05 PM Screenshot 2023-11-15 at 1 05 09 PM

its becoz 
BINARY, VARBINARY, TINYBLOB, BLOB, MEDIUMBLOB, LONGBLOB | VARBINARY
-- | --
in mysql.rst

File: presto-docs/src/main/sphinx/connector/postgresql.rst [hunk 2]
```diff
@@ -84,6 +84,106 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Type mapping
+------------
+
+PrestoDB and PostgreSQL each support types that the other does not. When reading from or writing to PostgreSQL, Presto converts
+the data types from PostgreSQL to equivalent Presto data types, and from Presto to equivalent PostgreSQL data types.
+
+PostgreSQL type to PrestoDB type mapping
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/postgresql.rst:93]:
```suggestion
PostgreSQL to PrestoDB type mapping
```
Change suggested to parallel https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst

File: presto-docs/src/main/sphinx/connector/postgresql.rst [hunk 3]
```diff
@@ -84,6 +84,109 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
...
+  * - ``UUID``
+    - ``UUID``
+  * - ``JSON``
+    - ``JSON``
+  * - ``JSONB``
+    - ``JSON``
+
+No other types are supported.
+
+PrestoDB type to PostgreSQL type mapping
```
[the remark this answers is not in the document]
Comment (steveburnett) [reply presto-docs/src/main/sphinx/connector/postgresql.rst:147]: This is not addressed yet. Change suggested to parallel https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst
Screenshot 2023-11-16 at 10 24 15 AM

File: presto-docs/src/main/sphinx/connector/postgresql.rst [hunk 4]
```diff
@@ -84,6 +84,106 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
...
+  * - ``DATE``
+    - ``DATE``
+  * - ``TIME``
+    - ``TIME``
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+  * - ``UUID``
+    - ``UUID``
+
+No other types are supported
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/postgresql.rst:185]:
```suggestion
No other types are supported.
```

**Review by steveburnett — CHANGES_REQUESTED** [review]
Nice, thanks! A few minor nits for consistency.

**Comment by Akanksha-kedia** [comment]
@steveburnett  please review

File: presto-docs/src/main/sphinx/connector/postgresql.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (steveburnett) [reply presto-docs/src/main/sphinx/connector/postgresql.rst:147]: This is still not addressed. Screenshot is from a build this morning. Before you mark a conversation as resolved, please build the doc locally and verify you've made the change. 
Screenshot 2023-11-20 at 10 22 43 AM

**Comment by Akanksha-kedia** [comment]
@steveburnett please review, i made the necessary chnages.

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

Thanks for updating the release notes entry!

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**